src/libraries/pkg/test/packageTest.targets

```diff
@@ -105,55 +132,27 @@
 
   <Target Name="VerifyNotDependsOnNetStandard"
           DependsOnTargets="ResolveReferences"
-          Condition="'$(_ShortFrameworkIdentifier)' == 'net' AND '$(_TargetFrameworkVersionWithoutV)' &gt;= '4.6.1' AND '$(RuntimeIdentifier)' == '' AND '$(DisableVerifyNotDependsOnNetStandardTest)' != 'true'">
+          Condition="'$(_ShortFrameworkIdentifier)' == 'net' AND '$(_TargetFrameworkVersionWithoutV)' &gt;= '4.6.1' AND '$(DisableVerifyNotDependsOnNetStandardTest)' != 'true'">
     <Error Condition="'$(DependsOnNetStandard)' == 'true'" Text="Package $(TestPackageId) requires netstandard shims when targeting $(TargetFramework)" />
   </Target>
 
   <PropertyGroup>
-    <_testDependsOn>
+    <TestDependsOn Condition="'$(TargetFramework)' != ''">
       LogBeginTest;
       VerifyNotDependsOnNetStandard;
       VerifyReferenceClosure;
       VerifyReferenceTypes;
       VerifyRuntimeClosure;
       VerifyRuntimeTypes;
-    </_testDependsOn>
+    </TestDependsOn>
+    <TestDependsOn Condition="'$(TargetFramework)' == ''">
```

Comment:
should we use ```Condition="'$(IsCrossTargetingBuild)' == 'true'"```  instead ?

Reply:
Will do this based on what we decide for https://github.com/dotnet/runtime/pull/53575#discussion_r644113768